base_search_custom_field_filter/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,5 @@
+This module allows to define custom filters in the search views for an specific
+field belonging to the document or any other related document.
+
+This nature makes the definition quite technical, but once done, it adds the
```

Comment:
you can probably drop this last sentence

Reply:
Why? I want to highlight that. Remember this is general "functional user" README, and a warning about this is always advisable.